.envrc

```diff
@@ -4,3 +4,6 @@ if has nix; then
   watch_file nix/{devShell,package,wraptest}.nix
   use flake
 fi
+
+# Source an `.envrc.local` if it exists. You can add your custom changes there.
+[[ -f .envrc.local ]] && source_env .envrc.local
```

Comment:
There is [source_env_if_exists](https://direnv.net/man/direnv-stdlib.1.html#codesourceenvifexists-ltfilenamegtcode) for this case, too.